core/lib/vm_interface/src/types/tracer.rs

```diff
@@ -37,3 +41,78 @@ pub enum VmExecutionStopReason {
     VmFinished,
     TracerRequestedStop(TracerExecutionStopReason),
 }
+
+/// Transaction validation parameters.
+#[derive(Debug, Clone)]
+pub struct ValidationParams {
```

Comment:
I'm not sure this belongs to the `tracer` module. Logically, I think validation doesn't necessary correlate to VM tracing (hence, the separate `TransactionValidator` trait).